sqlalchemy_bigquery/base.py

```diff
@@ -523,7 +528,8 @@ def visit_bindparam(
 
         if bindparam.expanding:
             assert_(self.__expanded_param(param), f"Unexpected param: {param}")
-            param = param.replace(")", f":{bq_type})")
+            if self.__sqlalchemy_version_info < (1, 4, 27):
```

Comment:
Looks like we're missing unit test coverage of entering the `if bindparam.expanding` if statement, but SQLAlchemy >= 1.4.27

```
sqlalchemy_bigquery/base.py                481      0    172      1    99%   531->541
```

I would have thought that the update to `setup.py` would make sure the tests run, though. 🤔